api/src/main/resources/workbench-api.yaml

```diff
@@ -4450,6 +4450,9 @@ definitions:
       complianceTrainingHost:
         type: string
         description: The hostname for constructing the compliance training URL.
+      complianceTrainingRenewalLookback:
```

Comment:
@evrii what do you think about a small followup PR for this change: group this one with `accessRenewalLookback` and update the description(s) to make the commonality and differences clearer.